.github/workflows/cron-checks.yml

```diff
@@ -43,27 +43,17 @@ jobs:
           - ios: 17.4
             xcode: 15.4
             os: macos-14
-            device: "iPhone 15 Pro"
+            device: "iPhone 14 Pro"
```

Comment:
Why do we change the devices? (same for below)

Reply:
Actually it's not required, just wanted to simplify things and use the same simulator where possible. I can return `iPhone 15 Pro` to `iOS 17.4` if you think it's needed.

Reply:
Just to confirm, it's not the Snapshot tests, it does not make a big difference here.